ruler/app.py

```diff
@@ -137,6 +137,7 @@ def create(event, context):
             "UserInfoEndpoint": oidc_user_info_endpoint,
             "ClientId": oidc_client_id,
             "ClientSecret": client_secret,
+            "Scope": "openid view download modify",
```

Comment:
Previously this was undefined, which is assumed to be the `openid` scope only, IIRC, which is only identity

Reply:
Yep. Documentation:
https://docs.aws.amazon.com/elasticloadbalancing/latest/APIReference/API_AuthenticateOidcActionConfig.html

Reply:
> This new scope allows view/download/modify on all of the user's Synapse data, right? Is it possible to scope to specific data sets?

I don't believe we currently support that in Synapse.

Reply:
OAuth scope does not let us specific specific data sets or projects. (There's a reason for this!) The listed scopes are correct. In particular they omit `offline_access` and `authorize` which is correct.  The complete list is here: https://rest-docs.synapse.org/rest/org/sagebionetworks/repo/model/oauth/OAuthScope.html